components/cluster/command/deploy.go

```diff
@@ -94,360 +61,46 @@ func newDeploy() *cobra.Command {
 				return nil
 			}
 
-			logger.EnableAuditLog()
```

Comment:
Is it by design?

Reply:
set in root.go PersistentPreRunE instead of every command setting this.